src/cosmicqc/analyze.py

```diff
@@ -95,9 +168,85 @@ def find_outliers(
     return outliers_df[columns_to_include]
 
 
+def label_outliers(
+    df: pd.DataFrame,
+    feature_thresholds: Optional[Union[Dict[str, float], str]] = None,
+    feature_thresholds_file: Optional[str] = DEFAULT_QC_THRESHOLD_FILE,
+    include_threshold_scores: bool = True,
+) -> pd.DataFrame:
+    """
+    This function uses z-scoring to format the data for detecting outlier
+    nuclei or cells using specific CellProfiler features. Thresholds are
+    the number of standard deviations away from the mean, either above
+    (positive) or below (negative). We recommend making sure to not use a
+    threshold of 0 as that would represent the whole dataset.
```

Comment:
I think this might be a bit more accurate:

```suggestion
Use identify_outliers to label the original dataset for where a cell passed or failed the quality control condition(s).
```

Reply:
Thanks, this adds greater clarity, I'll add this in!